devops/openshift/queue-consumers-deploy.yml

```diff
@@ -261,13 +261,30 @@ objects:
   - apiVersion: route.openshift.io/v1
     kind: Route
     metadata:
-      name: ${NAME}
+      name: ${NAME}-route
+      annotations:
+        haproxy.router.openshift.io/balance: leastconn
+        haproxy.router.openshift.io/disable_cookies: "true"
+        haproxy.router.openshift.io/hsts_header: max-age=31536000;includeSubDomains;preload
+        haproxy.router.openshift.io/timeout: 30s
```

Comment:
Isn't this the default value?

![image](https://github.com/user-attachments/assets/754e05b0-8fd0-4a8a-902b-c7e7a1d08527)

Reply:
Followed the agreement in the ticket. I would be ok to have it removed.